.env.example

```
@@ -21,6 +21,14 @@ FC_LLM_API_MODEL=gpt-3.5-turbo
 FC_LLM_API_TYPE=openai
 FC_DEFAULT_TARGET_LANG=zh-CN
 
+# Embedding Configuration (for embedding-filter feature)
+# If not set, falls back to LLM configuration above
+FC_EMBEDDING_API_TYPE=openai
+FC_EMBEDDING_API_BASE=https://api.openai.com/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FC_EMBEDDING_API_BASE key should go before the FC_EMBEDDING_API_TYPE key

(UnorderedKey)